src/Websites/Websites/Cmdlets/Certificates/NewAzureWebAppSSLBinding.cs

```diff
@@ -126,13 +126,12 @@ protected override void ProcessRecord()
                         pfxBlob: certificateBytes,
                         password: CertificatePassword,
                         hostingEnvironmentProfile: (webapp.HostingEnvironmentProfile != null) ?
```

Comment:
@panchagnula any reason that we can't change this line to just be `hostingEnvironmentProfile: webapp.HostingEnvironmentProfile` ?

Reply:
hostingProfileEnvironment is set only for Apps On an ASE, in other cases this is null

Reply:
To answer your question, yes we can